docs/build/group-chat.md

```diff
@@ -696,11 +706,15 @@ stream.stop()
 
 ## Sync group chats
 
-Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database. Be sure to periodically synchronize each group chat to ensure your app has the latest group chat details, including the most recent messages, member list, and group chat details, for example.
+Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database.
+
+Sync group chats anytime a user accesses a conversation view. This sync is especially important if a user backgrounds (switches away from) your app, then reopens the app and accesses a conversation view. 
+
+When the user reopens the app, the app might be using an old epoch (snapshot) of data. This is even more likely to be true if there were many group membership changes during the time the app was backgrounded. A group chat sync bumps the app to the latest epoch, helping to ensure that your user always sees the current group chat, including the most recent messages, member list, and group chat details, for example.
 
 Updates are also retrieved and added to the local database when streaming and when the user takes an action.
```

Comment:
I am unclear on what this sentence above means. Is this saying that streaming and user actions automatically trigger "sync group chat"?

Reply:
Not an automatic sync but streaming does receive messages and add them to local users database. My understanding is if a commit message is streamed (adding a user to the group) it should update the epoch in the stream.